src/AgConfigure.h

```diff
@@ -8,31 +8,6 @@
 
 class Configuration : public PrintLog {
```

Comment:
There is probably a better solution for the inheritance here, maybe create a new issue to look into this.